### ISSUE TITLE
Clear All Not working

### ISSUE THREAD

**Issue by Navjeet** [kdrtinchuk/Shopify-filter-2.0#1]
Hi, Thanks for the detailed code. Can you please check as clear all button is not working. Also it will be good if you can add sort by functionality as well.

Thanks

**Comment by kaushaltanchak** [comment]
I have changed the file main-collection. Please check it.
I hope you have to fix your issue as per my changes.

**Comment by Navjeet** [comment]
Yes I have fixed that, I have seen in the code that you have added the range slider for price filter, I have added the jquery and jquery UI js as well but the price range slider is not appearing. Is there any fix for this?
